src/store/Modules/Projects.ts

```diff
@@ -617,11 +622,12 @@ export default class Projects extends VuexModule {
       this.context.commit("updateQuery", data)
     } catch (err) {
       console.log(err)
+      this.context.commit("updateQuery", {})
```

Comment:
Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.